src/main/java/com/github/kongchen/swagger/docgen/reader/JaxrsReader.java

```diff
@@ -194,6 +198,37 @@ public String value(){
         return filteredMethods;
     }
 
+    private void readCommonParameters(Class<?> cls) {
+        Path path = AnnotationUtils.findAnnotation(cls, Path.class);
+        if (path != null) {
+            return;
+        }
+
+        List<Method> filteredMethods = getFilteredMethods(cls);
+        for (Method method : filteredMethods) {
+            path = AnnotationUtils.findAnnotation(method, Path.class);
+            if (path != null) {
+                return;
+            }
+
+            String httpMethod = extractOperationMethod(null, method, SwaggerExtensions.chain());
+            if (httpMethod != null) {
+                return;
+            }
+        }
+
+        Field[] fields = cls.getDeclaredFields();
+        for (Field field : fields) {
+            Annotation[] annotations = field.getAnnotations();
+            if (annotations.length > 0) {
+                List<Parameter> params = getParameters(cls, Arrays.asList(annotations));
+                for (Parameter param : params) {
+                    swagger.addParameter(param.getName(), param);
```

Comment:
Currently, if a parameter already exists with the same name, it is simply overwritten. This could lead to unexpected behaviour, since it is unclear which parameter will be in the generated swagger.json in the end.
We may want to stop the build at this point and inform the user about this.
What do you think?